File: Frontend/src/app/viewModels/responses/ListRoutesVM.ts
import { RouteResumeType } from "../Types/RouteResumeType";

/**
 * Represents a resumed route list, that is, without the locals information 
 */
export class ListRoutesVM {

    /**
     * Construct's an instance of {@link ListRoutesVM}
     * 
     * @param routes list of routes
     */
    constructor(
        public routes: Array<RouteResumeType>
    ) { }
}

File: Backend/Data/ViewModels/Response/ListRoutesVM.cs
using Backend.Data.ViewModels.Types;

namespace Backend.Data.ViewModels.Response
{
    /// <summary>
    /// View model class for the list routes response
    /// </summary>
    public class ListRoutesVM
    {
        public List<RouteResumeType> Routes { get; set; }

        /// <summary>
        /// Class' constructor method
        /// </summary>
        /// <param name="routes">List of routes</param>
        public ListRoutesVM(List<RouteResumeType> routes)
        {
            this.Routes = routes;
        }
    }
}